wp-content/themes/core/components/pagination/css/pagination.pcss

```diff
@@ -28,3 +28,44 @@
 
 	}
 }
+
+/*
+Pagination Single
+ */
+
+.pagination--single {
+	max-width: var(--grid-8-col);
+	margin: var(--spacer-60) auto;
+	border-top: 1px solid var(--color-border);
+	border-bottom: 1px solid var(--color-border);
+	padding: var(--spacer-40) 0;
+}
+
+.pagination__list {
+	display: flex;
+	flex-flow: row nowrap;
+	justify-content: space-between;
+	gap: var(--grid-gutter);
```

Comment:
another gap item to look at in Safari. Looks like current browser supports it but whats our standards on 1 version back?

Reply:
Looks like `gap` still needs to be removed in favor of margin.